content-directory-schemas/scripts/inputSchemasToEntitySchemas.ts

```diff
@@ -47,6 +44,7 @@ const ReferencePropertyDef = ({ Reference: ref }: ReferenceProperty): JSONSchema
   'oneOf': [
     onePropertyObjectDef('new', { '$ref': `./${ref.className}Entity.schema.json` }),
     onePropertyObjectDef('existing', { '$ref': `../entityReferences/${ref.className}Ref.schema.json` }),
+    PRIMITIVE_PROPERTY_DEFS.definitions.Uint64 as JSONSchema7,
```

Comment:
I'm guessing this is the main change needed to reference existing entitites in the chain?